src/lib/references.ts

```diff
@@ -176,6 +176,7 @@ export async function resolveContentRef(
                 href: targetSpace.urls.published ?? targetSpace.urls.app,
                 text: targetSpace.title,
                 active: true,
+                emoji: targetSpace.emoji
```

Comment:
`space.emoji` is a pretty internal thing to show on published docs. I'm not sure it's what users would expect.

At the very least they should expect to see the logo they configure in customization to take precedence over the emoji.

You can see how we decide to render the space logo in `LogoFallback` in `HeaderLogo` and I think we should use the same logic here. I'd argue that the emoji shouldn't be shown at all (falling back to the GitBook logo just like HeaderLogo does)

Reply:
Sure, I added `targetSpace.urls.logo` but I'll take a look at the logic in `LogoFallback` instead. Thanks for the pointer!

Reply:
Using the customization icon instead now. And resolving it in the component.